graylog2-server/src/main/resources/swagger/lib/swagger.js

```diff
@@ -305,7 +305,7 @@
           headers: {},
           on: {
             error: function(response) {
-              return _this.api.fail("Unable to read api '" + _this.name + "' from path " + _this.url + " (server returned " + error.statusText + ")");
+              return _this.api.fail("Unable to read api '" + _this.name + "' from path " + _this.url + " (server returned " + error.status + ")");
```

Comment:
this should be `response.status`

Reply:
Well, that's a facepalm. I corrected it, probably not in the workflow github envisioned for this, but its there anway.